packages/create-plugin/templates/common/.config/README.md

```diff
@@ -139,3 +139,26 @@ We need to update the `scripts` in the `package.json` to use the extended Webpac
 -"dev": "webpack -w -c ./.config/webpack/webpack.config.ts --env development",
 +"dev": "webpack -w -c ./webpack.config.ts --env development",
 ```
+
+### Configure grafana image to use when running docker
```

Comment:
Nice! 👏 

Could we also copy this to the [advanced config](https://github.com/grafana/plugin-tools/blob/main/docusaurus/docs/advanced-configuration.md) for the documentation website please?

Reply:
Done